src/App.tsx

```diff
@@ -10,31 +11,31 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const getselectedTabId = (id: string) => {
```

Comment:
if your function names start with "get" the expected behavior will be if the function returns smth - it means you will get some data. Your function only sets some value for the state, rename it please